packages/inter-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -398,15 +398,14 @@ test('vaultFactory display collateral', async t => {
     liquidationMargin: makeRatio(105n, run.brand),
     stabilityFee: makeRatio(530n, run.brand, BASIS_POINTS),
     marketPrice: makeRatio(5_555_555n, run.brand, 1_000_000n, aeth.brand),
-    interestRate: makeRatio(100n, run.brand, 10000n, run.brand),
```

Comment:
should the `stabilityFee` value be `100n` instead of `530n`?

Reply:
Looks like there was a pre-existing `stabilityFee` here colliding with the newly named `stabilityFee` (`interestRate`). It should have been the `mintFee` instead. Fixed.